1.8/development/standards.md

```diff
@@ -380,22 +380,28 @@ $db->insert_query('table', $new_record);
 
 Update queries follow a similar structure to insert queries, but use `update_query()` instead. This method takes a table name, an associative array of values to update, and optionally any conditions that must be matched (the `WHERE` clause of an update query).
 
-Values *must* be escaped and any conditions need to be explicitly escaped using the `escape_string()` method. Example use is as follows:
+Values *must* be escaped using the `escape_string()` method. Example use is as follows:
 
 {% highlight php startinline %}
 $updated_record = array(
     "column" => $db->escape_string('Value')
 );
+{% endhighlight %}
+
+Depending on the type of conditions it might also be required to use the `escape_string()` method, as mentioned above. However this would not be true for most cases. Both methods are shown here:
+
+{% highlight php startinline %}
+$db->update_query('table', $updated_record, "condition1='".$db->escape_string('randomvalue')."'");
 
-$db->update_query('table', $updated_record, "condition1='".$db->escape_string('test')."'");
+$db->update_query('table', $updated_record, "condition1=$randomvalue");
```

Comment:
... while you're not escaping the variable, which is more likely to contain disallowed characters. But as I said, the best example for a string that needs to be escaped is `$mybb->get_input('randomstringinput')`